apps/passport-server/migrations/71_generic_issuance_semaphore_history.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE generic_issuance_semaphore_history (
+  id BIGSERIAL PRIMARY KEY,
```

Comment:
upon further reflection it probably does - a group can have the same group root at different points in time, so some sort of total order needs to be enforced.

Reply:
This is copied from the existing semaphore_group_history table. The id auto-increments, so the higher number is always more recent, and this is how we know which is the most recent history for a group. This might not be the most elegant way to solve that problem, but having per-group sequences would probably involve having a postgres trigger function which runs on each insert to increment a counter, which is more complex.